src/Microsoft.AspNetCore.SystemWebAdapters/SystemWebAdapterExtensions.cs

```diff
@@ -2,47 +2,38 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System;
-using System.Collections;
-using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
 using System.Web;
 using Microsoft.AspNetCore.Http.Features;
-using Microsoft.AspNetCore.SystemWebAdapters.Internal;
 
 namespace Microsoft.AspNetCore.SystemWebAdapters;
 
-internal static class InternalSystemWebExtensions
+public static class SystemWebAdapterExtensions
```

Comment:
Do we make the method names more specific? Seeing GetAdapter looks a little hard to understand. 

-  `GetHttpRequestAdapter()`
- `GetSystemWebRequest()`

Feels API review worthy.

Reply:
reviewed and updated